### PR TITLE
94 check detector selection exist

### LINKED ISSUE #87: Don't auto-plot the first file
@apsIEX think it is a bad idea.

### LINKED ISSUE #94: When switching file, check if previous detector selection exist for new file
Since PR #91, when closing #41, we did not address the case where one or more of the detectors selected do not exist in the next file. Cannot just rely on the table view row, need to check PV name,

### PR DESCRIPTION
**Pull request by rodolakis**
- close #94 
- close #87

### CODE REVIEW

File: mdaviz/chartview.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def addCurve(self, row, *args, **kwargs):
         # Add to the dictionary
         label = kwargs.get("label", None)
         self.line2D[label] = plot_obj[0], args[0], args[1], row
-        print(f"Calling addCurve: row={self.line2D[label][3]}")
+        # print(f"Calling addCurve: row={self.line2D[label][3]}")
```
Comment (MDecarabas) [inline mdaviz/chartview.py:158]: There are a lot of print statements commented out, do we need them?

**Review by MDecarabas — APPROVED** [review]
LGTM overall! Played around with it and the PR addresses the issues.

**Comment by MDecarabas** [comment]
Traceback (most recent call last):
  File "/Users/ecodrea/bluesky/mdaviz/mdaviz/select_fields_table_model.py", line 173, in setData
    self.setCheckbox(index, value)
  File "/Users/ecodrea/bluesky/mdaviz/mdaviz/select_fields_table_model.py", line 204, in setCheckbox
    self.checkboxStateChanged.emit(self.plotFields()[0])
                                   ^^^^^^^^^^^^^^^^^
  File "/Users/ecodrea/bluesky/mdaviz/mdaviz/select_fields_table_model.py", line 379, in plotFields
    choices[column_name].append(row)
    ~~~~~~~^^^^^^^^^^^^^
KeyError: 'Norm'


Lots of errors if you change the variables in the second table

File: mdaviz/chartview.py (hunk 1 above)
Comment (rodolakis) [reply mdaviz/chartview.py:158]: probably not, let me removed them

**Comment by rodolakis** [comment]
> Traceback (most recent call last): File "/Users/ecodrea/bluesky/mdaviz/mdaviz/select_fields_table_model.py", line 173, in setData self.setCheckbox(index, value) File "/Users/ecodrea/bluesky/mdaviz/mdaviz/select_fields_table_model.py", line 204, in setCheckbox self.checkboxStateChanged.emit(self.plotFields()[0]) ^^^^^^^^^^^^^^^^^ File "/Users/ecodrea/bluesky/mdaviz/mdaviz/select_fields_table_model.py", line 379, in plotFields choices[column_name].append(row) ~~~~~~~^^^^^^^^^^^^^ KeyError: 'Norm'
> 
> Lots of errors if you change the variables in the second table

I have not implemented `Norm` nor `Mon`, so yeah if you click on any of those it is going to crash. I should probably either hide them for now or make sure I handle that case to prevent crashing. Created issue #97

File: mdaviz/chartview.py (hunk 1 above)
Comment (rodolakis) [reply mdaviz/chartview.py]: Done

**Pull request merged**